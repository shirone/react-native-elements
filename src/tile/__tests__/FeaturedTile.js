import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import FeaturedTile from '../FeaturedTile';

describe('FeaturedTitle component', () => {
  it('should render without issues', () => {
    const component = shallow(<FeaturedTile imageSrc={{url: 'http://google.com'}}/>);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
