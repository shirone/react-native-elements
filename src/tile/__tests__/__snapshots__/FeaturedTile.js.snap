// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeaturedTitle component should render without issues 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  focusedOpacity={0.7}
  style={
    Array [
      Object {
        "height": 600,
        "width": 750,
      },
      undefined,
    ]
  }
>
  <Image
    source={
      Object {
        "url": "http://google.com",
      }
    }
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "height": 600,
          "justifyContent": "center",
          "resizeMode": "cover",
          "width": 750,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "rgba(0,0,0,0.2)",
            "bottom": 0,
            "flex": 1,
            "justifyContent": "center",
            "left": 0,
            "paddingBottom": 40,
            "paddingLeft": 25,
            "paddingRight": 25,
            "paddingTop": 45,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "justifyContent": "center",
            },
            undefined,
          ]
        }
      />
      <TextElement
        h4={true}
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,0)",
              "color": "#ffffff",
              "marginBottom": 15,
              "textAlign": "center",
            },
            undefined,
          ]
        }
      />
      <TextElement
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,0)",
              "color": "#ffffff",
              "marginBottom": 15,
              "textAlign": "center",
            },
            undefined,
          ]
        }
      />
    </View>
  </Image>
</TouchableOpacity>
`;
